src/index.js

```diff
@@ -49,11 +43,10 @@ function whichTripsToDisplay(tripStatus) {
 
 
 function clickLog() {
-    console.log("allDestinations", allDestinations)
-    // console.log("userData", userData)
-    // whichTripsToDisplay('approved');
-    // console.log(tripsToDisplay)
+    // console.log("allDestinations", allDestinations)
+    console.log("user", user)
+    console.log(user.trips)
 }
 
-// export default userData;
-export default userStuff;
+export default fetchSetter;
+// maybe rename userStuff, maybe 
```

Comment:
You _did_ rename userStuff...stop living in the past. It's time to let go...

Reply:
Be gone!